tests/Doctrine/AnnotationTest.php

```diff
@@ -11,7 +11,7 @@ class AnnotationTest extends AbstractTest
     public function testAnnotation(): void
     {
         self::markTestIncomplete();
-        $documentManager = $this->getDocumentManager();
-        $metadata        = $documentManager->getClassMetadata('DoctrineMongoODMModuleTest\Assets\Document\Annotation');
+        //$documentManager = $this->getDocumentManager();
+        //$metadata = $documentManager->getClassMetadata('DoctrineMongoODMModuleTest\Assets\Document\Annotation');
```

Comment:
Why keeping the commented code?

Reply:
Well, I don't know what this test was supposed to do. It obviously didn't do anything in the last years, because of the `self::markTestIncomplete()` line above.

Since there are no other tests in that file, the whole file is somehow obsolete.

Reply:
I'm actually thinking about removing this test. It seems like it lost it's meaning during development.

Reply:
Ok, I removed it with 7e23850